Ship_Game/UI/UIButton_Style.cs

```diff
@@ -36,13 +37,13 @@ public class StyleTextures
 
             // Text Colors
             public Color DefaultTextColor = new Color(255, 240, 189);
-            public Color HoverTextColor   = new Color(255, 240, 189);
-            public Color PressTextColor   = new Color(255, 240, 189);
+            public Color HoverTextColor = new Color(255, 240, 189);
+            public Color PressTextColor = new Color(255, 240, 189);
```

Comment:
Please restore previous formatting here

Reply:
restored in commit "reverted formatting"